kodilla-testing/src/test/java/com/kodilla/testing/collection/CollectionTestSuite.java

```diff
@@ -0,0 +1,50 @@
+package com.kodilla.testing.collection;
+
+import org.junit.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class CollectionTestSuite {
+    private OddNumbersExterminator oddNumbersExterminator;
+
+    @Before
+    public void before(){
+        System.out.println("Test Case (collection): begin");
+    }
+    @After
+    public  void after(){
+        System.out.println("Test Case (collection): end");
+    }
+    @BeforeClass
+    public static void beforeClass() {
+        System.out.println("Test Suite (collection): begin");
+    }
+    @AfterClass
+    public static void afterClass() {
+        System.out.println("Test Suite (collection): end");
+    }
+    @Test
+    public void testOddNumbersExterminatorEmptyList(){
+        //Given
+        oddNumbersExterminator = new OddNumbersExterminator();
```

Comment:
Nie do konca o to chodzilo 
`private OddNumbersExterminator oddNumbersExterminator = new OddNumbersExterminator();`
w lini 13. Dzieki temu pozbywasz sie Given w obu testach

Reply:
OK

Reply:
Pytanie czy przykłady z sub modułów poprawiać czy traktować tylko jako przykład.... jeżeli chodzi  o działanie...

Reply:
pytanie czy w drugim teście w (//given ) może  być tworzenie testowej listy czy ma być przeniesiona do (//when) gdzie jest tworzona oczekiwana lista wyniku działania testowanej metody